source/object-model/classes-and-instances.md

```diff
@@ -113,11 +113,10 @@ Person.create({
 // alerts "Stefan Penner, reporting for duty!"
 ```
 
-If you are subclassing a framework class, like `Ember.View` or
-`Ember.ArrayController`, and you override the `init` method, make sure
-you call `this._super()`! If you don't, the system may not have an
-opportunity to do important setup work, and you'll see strange behavior
-in your application.
+If you are subclassing a framework class, like `Ember.View`, and you override
```

Comment:
I thought about that, but from looking in `Ember.Controller`, I didn't see that it had an `init` method. I think a subsequent PR should remove references to `Ember.View`, but I'm not sure of how to handle this paragraph.